server/src/main/java/com/lunchtime/models/Order.java

```diff
@@ -0,0 +1,83 @@
+package com.lunchtime.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.ColumnDefault;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.time.Instant;
+import java.util.Date;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+@Data
+@Table(name = "ordering")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "person_id", referencedColumnName = "id")
+    private Person person;
+
+    @Column(name = "waiter_id")
+    private Long waiter;
+
+    @NotNull
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "start_time")
+    private Date startTime;
+
+    @NotNull
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "finish_time")
+    private Date finishTime;
```

Comment:
As I discovered `@Temporal` annotation by default sets `TemporalType.TIMESTAMP`. Anyway, I can't understand clearly its benefit. Could you explain simply the purpose of `@Temporal` annotation here and how this is used in rest part of code now? As I understand, this is for converting incoming date from UI to sql type in DB?

Reply:
I think, it converts Java Date to Database Date. I can't explain more :(

Reply:
Ok, thanks